app/serializers/publishing_house_serializer.rb

```diff
@@ -1,4 +1,8 @@
 class PublishingHouseSerializer < ActiveModel::Serializer
   attributes :id, :name, :discount
   has_many :published
+
+  def discount
+    object.discount.to_f
```

Comment:
Wanted to make sure the discount is with the format that i want to expose for my API consumer

Reply:
Agreed